tests/Api/Shop/CheckoutCompletionTest.php

```diff
@@ -278,6 +278,20 @@ private function getFirstPaymentId(string $tokenValue): string
             uri: sprintf('/api/v2/shop/orders/%s', $tokenValue),
         );
 
-        return (string) json_decode($this->client->getResponse()->getContent())->payments[0]->id;
+        $content = $this->client->getResponse()->getContent();
+        Assert::notFalse($content);
```

Comment:
Just pointing out, that `tests/Api/Admin/OrdersTest.php` uses new dependency `use PHPUnit\Framework\Assert;`, while this class uses webmozart as usual.

Is PHPUnit assert needed?

Reply:
This is just to satisfy static analysis

Reply:
Okay, initially I missed your point. I've corrected the namespace